library/src/main/java/com/bumptech/glide/load/data/StreamLocalUriFetcher.java

```diff
@@ -12,17 +19,78 @@
  * Fetches an {@link java.io.InputStream} for a local {@link android.net.Uri}.
  */
 public class StreamLocalUriFetcher extends LocalUriFetcher<InputStream> {
+    /** A lookup uri (e.g. content://com.android.contacts/contacts/lookup/3570i61d948d30808e537) */
+    private static final int ID_LOOKUP = 1;
+    /** A contact thumbnail uri (e.g. content://com.android.contacts/contacts/38/photo) */
+    private static final int ID_THUMBNAIL = 2;
+    /** A contact uri (e.g. content://com.android.contacts/contacts/38) */
+    private static final int ID_CONTACT = 3;
+    /**
+     * A contact display photo (high resolution) uri
+     * (e.g. content://com.android.contacts/5/display_photo)
+     */
+    private static final int ID_DISPLAY_PHOTO = 4;
+
+
+    private static final UriMatcher URI_MATCHER;
```

Comment:
Name should include `Contact(s)`, because it's specific to that; if it was extended in the future the host/authority wouldn't match? Or leave it as is, but change the matching logic to decide solely on this and not the host (doesn't the UriMatcher start with looking up the host in an authority hash anyway?)